src/hooks/useKeyboardShortcut.ts

```diff
@@ -0,0 +1,56 @@
+import {useEffect} from 'react';
+import {ValueOf} from 'type-fest';
+import KeyboardShortcut from '@libs/KeyboardShortcut';
+import CONST from '@src/CONST';
+
+type Shortcut = ValueOf<typeof CONST.KEYBOARD_SHORTCUTS>;
+type KeyboardShortcutConfig = {
+    captureOnInputs?: boolean;
```

Comment:
@kubabutkiewicz Maybe you can add comments, you can reuse [this](https://github.com/Expensify/App/blob/d7b14792974b4082ab574e7772dc8f036ffe38b0/src/libs/KeyboardShortcut/index.ts#L124) I guess